tests/benchmark/apis_test.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package benchmark
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/rpc"
+	"github.com/google/uuid"
+)
+
+const (
+	rootResource = "projects/bench/locations/global"
```

Comment:
We might occasionally need to use a different value for this. Can the tests take arguments? We could also set it with an environment variable (that we replace with a default value if unset)

Reply:
I agree with the `APG_REGISTRY_PROJECT` suggestion. We need something similar for the [registry-viewer](https://github.com/apigee/registry-viewer).